clar2wasm/src/standard/standard.wat

```diff
@@ -228,7 +228,7 @@
     )
 
     ;; This function can be used to add either signed or unsigned integers
-    (func $add-int128 (type 1) (param $a_lo i64) (param $a_hi i64) (param $b_lo i64) (param $b_hi i64) (result i64 i64)
+    (func $stdlib.add-int128 (type 1) (param $a_lo i64) (param $a_hi i64) (param $b_lo i64) (param $b_hi i64) (result i64 i64)
```

Comment:
There is no point renaming those functions, we can just rename the exports.

Reply:
Apparently I was very wrong here, my bad :D

Reply:
No problem @Acaccia ! Code review is to tackle things like that.